awx/ui_next/src/screens/Inventory/InventoryDetail/InventoryDetail.jsx

```diff
@@ -76,9 +76,11 @@ function InventoryDetail({ inventory, i18n }) {
           value={
             <ChipGroup numChips={5}>
               {instanceGroups.map(ig => (
-                <Chip key={ig.id} isReadOnly>
-                  {ig.name}
-                </Chip>
+                <div css="padding-right: 5px" key={ig.id} isReadOnly>
```

Comment:
Link cannot be a child of Chipgroup(children must be an li), so I had to add a div in there.  Also, ideally if/when patternfly adds a label group component they will handle the spacing also.